desktop/core/src/desktop/api2_tests.py

```diff
@@ -37,6 +37,113 @@ def setUp(self):
     self.user = User.objects.get(username="api2_user")
 
 
+  def test_import_document_with_forward_ref(self, client=None):
+    if client is None:
+      client = self.client
+    
+    doc = '''[\n{\n  "model": "desktop.document2",\n  "pk": 20,\n  "fields": {\n    "owner": [\n      "admin"\n    ],\n'''\
```

Comment:
As in the other PR, please keep the minimal set of docs and fields that are needed for your test. It will make for simpler understanding and debugging later on.

Reply:
okay, I will change it.